point-of-sale-app/inventory/src/main/resources/test.config

```diff
@@ -0,0 +1,2 @@
+CONNECTOR=DATABASE
+ITEMS=items:\n  - name: "BigBurger"\n    type: "burgers"\n    price: 5.50\n    imageUrl: "usr/lib/images/bigburger.png"\n    quantity: 200\n    labels: [ "retail", "restaurant", "food" ]\n  - name: "DoubleBurger"\n    type: "burgers"\n    price: 7.20\n    imageUrl: "usr/lib/images/burgers.png"\n    quantity: 200\n    labels: [ "retail", "restaurant", "food" ]\n  - name: "Shirt"\n    type: "textile"\n    price: 15.50\n    imageUrl: "usr/lib/images/shirt.png"\n    quantity: 50\n    labels: [ "retail", "textile", "clothing" ]\n  - name: "Short"\n    type: "textile"\n    price: 17.20\n    imageUrl: "usr/lib/images/short.png"\n    quantity: 20\n    labels: [ "retail", "textile", "clothing" ]
```

Comment:
To avoid the whitespacing, could we use JSON instead of YAML here?

Reply:
The default case for loading these configurations is via a [`ConfigMap`](https://github.com/GoogleCloudPlatform/point-of-sale/blob/main/point-of-sale-app/kubernetes-manifests/service-configs.yaml#L26-L51). This sample is added only to help someone who wants to run the services locally (for development purposes) in a non-K8s setting. I would be deleting this and moving this sample to the README when I write one.